publishing nettoyer-ses-donnees-avec-openrefine lesson in FR

spapastamkou opened the pull request:
Text and FR image files for the FR translated lesson: Nettoyer ses données avec Openrefine (Cleaning Data with Openrefine).

spapastamkou committed:
text and image files of operefine lesson in FR

spapastamkou committed:
Update nettoyer-ses-donnees-avec-openrefine.md

changed translators to translator in yaml

spapastamkou committed:
Update nettoyer-ses-donnees-avec-openrefine.md

mdlincoln commented:
Will take a look at this tonight!

mdlincoln commented on fr/lecons/nettoyer-ses-donnees-avec-openrefine.md:
```diff
@@ -0,0 +1,157 @@
+abstract: "Ce tutoriel explique comment les chercheur(e)s peuvent déterminer l'état de précision de leurs données et comment agir pour l'améliorer." 
+original: /en/lessons/cleaning-data-with-openrefine
+review-ticket: https://github.com/programminghistorian/ph-submissions/issues/223 
+redirect_from: /fr/lecons/nettoyer-ses-donnees-avec-openrefine
```
No `redirect_from` is needed on any new lessons. You can delete this.

mdlincoln committed:
.keep no longer needed in french lessons folder

mdlincoln committed:
remove alert for upcoming french lessons

mdlincoln commented on fr/lecons/nettoyer-ses-donnees-avec-openrefine.md:
```diff
@@ -0,0 +1,157 @@
+activity: transforming
+topics: [data-manipulation]
+abstract: "Ce tutoriel explique comment les chercheur(e)s peuvent déterminer l'état de précision de leurs données et comment agir pour l'améliorer." 
+original: /en/lessons/cleaning-data-with-openrefine
```
Please note in the [translation guidelines](https://programminghistorian.org/fr/consignes-redacteurs#3-ajouter-les-m%C3%A9tadonn%C3%A9es-en-yaml-dans-le-fichier-de-la-le%C3%A7on) that `original` should only show the lesson slug, no other path elements, e.g.

```
original: cleaning-data-with-openrefine
```

mdlincoln commented on fr/lecons/nettoyer-ses-donnees-avec-openrefine.md:
```diff
@@ -0,0 +1,157 @@
+
+Le mot « valeur » dans le champ texte représente la valeur actuelle de chaque cellule, que vous pouvez voir ci-dessous. Nous pouvons modifier cette valeur en lui appliquant des fonctions (voir la [documentation GREL](https://github.com/OpenRefine/OpenRefine/wiki/GREL-Functions "documentation GREL") pour une liste complète). Dans ce cas, nous voulons remplacer les doubles **pipes** par un seul **p ...
+
+value.replace('||', '|')
```
either indent or wrap with triple backticks so that this renders as code

mdlincoln commented on fr/lecons/nettoyer-ses-donnees-avec-openrefine.md:
```diff
@@ -0,0 +1,157 @@
+- François Dominic Laramée
+- Marie Puren
+difficulty: 2
+exclude_from_check:
```
So this translation actually _does_ have a review ticket, unlike the old lesson, so please remove this entry from the YAML

spapastamkou replied:
just done this and previous requested changes

mdlincoln committed:
set to singular until the 2nd french lesson is published

mdlincoln committed:
check that lesson 'original' slug is only basname, no URL parts

spapastamkou committed:
Update nettoyer-ses-donnees-avec-openrefine.md

changes requested by matt

spapastamkou committed:
Update nettoyer-ses-donnees-avec-openrefine.md

added lesson slug in yaml header

spapastamkou committed:
Update nettoyer-ses-donnees-avec-openrefine.md

removed ":" from end of link

fdlaramee reviewed (changes requested):
A few things to look at:

Line 101: There is a singular/plural mismatch in the sentence beginning with "Expérimentez..." 
Line 115: No idea what that sequence of \* \* represents; may just be an encoding thing I'm unfamiliar with.
LIne 118: Might want to enclose in triple backticks to render as code, just like the similar segment a couple of lines earlier.

Thanks,
FDL

mdlincoln committed:
remove escaped asterisks

mdlincoln reviewed (changes requested):
Thanks for addressing those formatting changes.

Finally, you need to add translated bios for the authors.

mdlincoln commented on fr/lecons/nettoyer-ses-donnees-avec-openrefine.md:
```diff
@@ -0,0 +1,154 @@
+layout: lesson
+slug: nettoyer-ses-donnees-avec-openrefine
+date: 2013-08-05
+translation_date: 2018-11-27
```
Sorry, one more note - please update this date to `2019-04-10` so it's closer to the publication date

spapastamkou committed:
Update nettoyer-ses-donnees-avec-openrefine.md

modified date in yaml header - put all pipes between tripleticks - corrected gender accord (as per fdl and matt's comments)

spapastamkou committed:
Update ph_authors.yml

translated 3 authors bios in FR

mdlincoln committed:
escape pipes to prevent a table from being written

mdlincoln committed:
Merge branch 'nettoyer-ses-donnees-avec-openrefine-lesson' of github.com:programminghistorian/jekyll into nettoyer-ses-donnees-avec-openrefine-lesson

mdlincoln commented:
OK, it looks like you've gotten all the main points I could see. Please make sure to look at this preview render carefully to make sure all the text is rendering the way you want:

https://deploy-preview-1263--ph-dev.netlify.com/fr/lecons/nettoyer-ses-donnees-avec-openrefine

mdlincoln commented:
Go ahead and merge when ready @spapastamkou - and congratulations @programminghistorian/french-team !!

spapastamkou commented:
Ok, I will, @mdlincoln, I just need to fix some small problems before:)

spapastamkou committed:
Update nettoyer-ses-donnees-avec-openrefine.md

fixed some typos, spaces etc - added FR translation of CC licence - harmonised abstract with the main text (where the phrase came from)

spapastamkou commented:
I'll wait a bit to see what happens with #1265 and then merge.

spapastamkou committed:
Update nettoyer-ses-donnees-avec-openrefine.md

updated api url and added explicative note as per #1265
